### ISSUE TITLE
Files cannot be imported from TypeScript with module: NodeNext

### ISSUE THREAD

**Issue by SLaks** [hebcal/hebcal-leyning#432]
I'm trying to run unit tests in Node.js using https://github.com/avajs/ava and https://github.com/tapjs/tsimp.

tsimp forces tsconfig.json to specify `"moduleResolution": "NodeNext"`, which in turn gives the following errors:

```
Module '"@hebcal/leyning"' has no exported member 'Aliyah'.
ts(2305)

// Caused by

Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path.
ts(2834)
```

I think you can fix this by changing `index.d.ts` to add a file extension to its relative imports.

The same error applies to `@hebcal/core`, but it doesn't actually block me right now because I don't rely on types or forwarded imports from it.

**Comment by mjradwin** [comment]
Our `.d.ts` files are generated by typescript. I'm not sure there's an easy way to add a file extension.

**Comment by mjradwin** [comment]
OK - we're trying to fix this - can you let us know if this works? We released 8.2.6 which rolls up the .d.ts into a single file

**Comment by SLaks** [comment]
That works!

Thank you very much!
